### ISSUE TITLE
[BUG] page_entities not showing when room otherwise "empty"

### ISSUE THREAD

**Issue by patman15** [dwainscheeren/dwains-lovelace-dashboard#156]
Hi!

If rooms do not have any `light`/`device`/`cover`/etc. defined, `page_entities` are not shown. Here is a minimal example, like the one from the [documentation](https://dwainscheeren.github.io/lovelace-dwains-theme/configuration/rooms.html), that does not work:
```
  - name: Bedroom
    icon: fal:bed
    page_entities:
      columns: 1 #optional
      show_title: 'false' #optional
      entities:
        - entity: sensor.name1
        - entity: sensor.name2
```
I could trace it down to the fact that the `dwains-flexbox-card` does not like the fact that there are no `cards` defined. Unfortunately, I do not know how to debug it, but I think the card throws an exception and the remaining parts are thus not rendered.

Also, I was also wondering, why the `page_entities` are in a separate `dwains-flexbox-card`. Probably because of the heading? Patching `/dwains-theme/views/main/rooms/room.yaml` like in the following puts all items in one `dwains-flexbox-card` and also solves the above issue, as the card is not empty any more:
```
          #Start of extra room entities
          {% if room["page_entities"] and room["page_entities"]["entities"] %}
#          - type: custom:dwains-flexbox-card
#            items_classes: 'col-xs-12 col-sm-6 col-md-4 col-lg-3'
#            padding: true
#            cards:
#              {% if room["page_entities"]["show_title"] != 'false' %}
              #Heading
#              - !include
#                - ../../partials/heading.yaml
#                - title: {{ _d_t_trans.more_entities.title }}
#              {% endif %}
              {% for more_entity in room["page_entities"]["entities"] %}
              - type: custom:button-card
                entity: {{ more_entity["entity"] }}
                template: room_more_entity
                tap_action:
                  !include 
                    - ../../popups/more-info.yaml
                    - entity: {{ more_entity["entity"] }}
              {% endfor %}
              {% endif %}
```

**Comment by deku-m** [comment]
I have the same issue using page entities room not showing entities as i use it to put a switch entity in a room.
But it won't show.

**Comment by dwainscheeren** [comment]
Fixed in Dwains Dashboard 2.0
